project-base/app/config/private.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpQIBAAKCAQEA4Y3boig3D3Si/OxTJfGz2bm68lTydupL0f5vFU1pPfQ73eKR
+p7oFH2i5TNKtUY5Ys48Sr/iyv4s10YhjhouJmt/3OvaFD09yo07Vi7osoPqFZWIc
+84Z0lNmZxRjNhD5K3eIqKGBE9894mpSG4OoiWv610RvVdHOLNLq/tQtBWHwILPS/
+vkSn5pZxcBaLX285s5qGXskBLrbuU9oqZvfIArXxNhrVuGlhmMFKJ0cFOJ45OTtp
+ogBQYaLt0m8AN4RDE9Qy11t88mR2pLfHJlyx3gw2kh5itL9GkKsnfNS+zxh+2XqO
+G9oLsvtNskQEyjEaBXUX3r7B1lulHTibvnyl1QIDAQABAoIBAQCiDllincBJs4Sz
+clyfY5TT9zURjeUTeW12eHd5d8AIymUFHBeIygeKNKUGRExQz5R9lmtX0bkkFDJb
+1MGlVrZMd2RbkemyQWPdX/PgS6hrFPQpDqA8fDthAyMPUK3AuTzzRN8DWX/UCNXf
+Qp3SFYcPl/gsQPJRk/7NqOu6eVa+6i1qqMKwk/23+7JZYNK5m8cS9iPmYlItm60Q
+/S63E8bnrPSyHZxDp1HDdKsQePfSvRtYUPB4tePQew/32UOMAW5ur3sVaBgbFbMV
+ay9tFcfWlUJyVbhfYkfggUk7dtrvlayqZgydnGtkE4lYdwadeTfdCVYH5VEQFk8+
+9ZcqMcEVAoGBAPH6wL7sufFsKCkR5i6iw53QLfzgg92tikBVZ4tp6np7jDnBQTlE
+bXC1iaGwIdH6Z4X+bm6C09WkBAK50xljNM9y40poEs3WCxkrfjinjo/yhdIg55WA
+RIBfMyG4we2a9W7Mr2+PdOoQIXuR9LgyniF6UTLbvecbT3PosXq4SwMLAoGBAO6f
+eElspgJTj63GD/r+K2VakxLPyvjTl0eZoJyxWRofAElfcwi1RVWyaM8QxtPZ+3M0
+S/wFTybz+Psj9Ed83anlyt5I8AY017mcAfYYslLvjDF1am+ZmXf4c+tRSBb1y8IS
+AkI7S4CoKFM6JgoPV0/HtyBvIJ8dU1zh7aNdqoafAoGAdAld0nBsfkPyQjeOHQD6
+Mzw4QVJciboFWPJbc/Q3d8K1LKDqqQ6SOIElXb67iRi9+fhXdnEQKhu7vJ7lhwAW
+vAAI1I5xrfkdW1WK8poO+CJBX9FdnEhlEPYV6+7QOCOd3CMVYbb2RqOzmrEUHBxR
+zhX8yBmtyBxFjnygVwv3FJ8CgYEAwoxRzcs/LfZi1hL9RG2lg+TstBGmMmiQlley
+kw4kzx8nZKrlZtwjw1LKEMhk0qSOVcb1jGvntKKFBHyz1PUA5Fvton1k9CpxF2sS
+TkqkWOnlKw3H0WUxspZmLB0A4XdUwM1d+hHN5P80oWZvTRRk4y65J0BzIEcQiFHL
+Jb5gIZECgYEAtNeBia3WRDx5oWkgrQV30ykuOT5XMCaCSxzQkTCILCZ2mNbZ4pS1
+Lhx7/h1v3IC+xbSTIEtWwtoZB8K3HcUGqA/eMLx5kwp9TnM5W2AzTO6AouxiQD0E
+GzN9fI2yAxeS8A1DrRX/U/89p/swBT3E+GJ0/3KfTu9onaT/um1XpsY=
+-----END RSA PRIVATE KEY-----
```

Comment:
Please don't commit private keys. Ever. Just don't. Not ever. No. Please, no.

Reply:
@TomasLudvik ;-)